app/main/views/suppliers.py

```diff
@@ -0,0 +1,23 @@
+from flask import jsonify
+
+from .. import main
+from ...models import Supplier
+
+
+@main.route('/suppliers/<int:supplier_id>', methods=['GET'])
+def get_supplier(supplier_id):
+    supplier = Supplier.query.filter(
+        Supplier.supplier_id == supplier_id
+    ).first_or_404()
+
+    return jsonify(suppliers=jsonify_supplier(supplier))
+
+
+def jsonify_supplier(supplier):
+    data = {
+        'id': supplier.id,
```

Comment:
Yeah, I dunno.

![screen shot 2015-03-31 at 10 12 54](https://cloud.githubusercontent.com/assets/2454380/6915751/a9865f56-d78e-11e4-9fcf-b2059d90b89d.png)

Here's what my supplier table (with example data that I've added) looks like now.  It's possible to have a divergent `id` and the `supplier_id` if you create them manually, but if they're always meant to be identical than we probably don't need both.